draft-packages/illustration/KaizenDraft/Illustration/Scene.tsx

```diff
@@ -32,9 +32,9 @@ export const BrandMomentCaptureIntro = ({
     return (
       <VideoPlayer
         {...otherProps}
-        fallback="illustrations/heart/scene/brand-moments-capture-intro-loop.png"
-        ambientAnimation="illustrations/heart/scene/brand-moments-capture-intro-loop.webm"
-        initialAnimation="illustrations/heart/scene/brand-moments-capture-intro.webm"
+        fallback="illustrations/heart/scene/brand-moments-capture-intro-loop"
+        ambientAnimation="illustrations/heart/scene/brand-moments-capture-intro-loop"
+        initialAnimation="illustrations/heart/scene/brand-moments-capture-intro"
```

Comment:
It felt appropriate to allow VideoPlayer to choose the right file format depending on usage - so I've removed all file extensions.